app/components/wds-global-footer.js

```diff
@@ -3,4 +3,21 @@ import Component from '@ember/component';
 export default Component.extend({
 	tagName: 'footer',
 	classNames: 'wds-global-footer',
+
+	init() {
+		this._super(...arguments);
+
+		if(!this.track) {
+			throw new Error('Required `track` function for wds-global-footer component is not set');
+		}
+	},
+
+	click(event) {
+		const elementToTrack = event.target.closest('[data-tracking-label]');
```

Comment:
Would be nice to have a boundary to prevent leaking this from the footer element.